c/httpfileservice.c

```diff
@@ -413,37 +448,73 @@ void copyUnixDirectoryAndRespond(HttpResponse *response, char *oldAbsolutePath,
 static int copyUnixFile(char *oldAbsolutePath, char *newAbsolutePath, int forceCopy) {
   int returnCode = 0, reasonCode = 0, status = 0;
   FileInfo info = {0};
+  
+  if(!strcmp(oldAbsolutePath,newAbsolutePath)){
+    zowelog(NULL, LOG_COMP_RESTFILE, ZOWE_LOG_DEBUG,
+            "Invalid input, same file path (source=%s, destination=%s)\n",
+            oldAbsolutePath, newAbsolutePath);
+    return HTTP_FILE_SERVICE_INVALID_INPUT;    
+  }
 
   status = fileInfo(oldAbsolutePath, &info, &returnCode, &reasonCode);
   if (status == -1) {
     zowelog(NULL, LOG_COMP_RESTFILE, ZOWE_LOG_DEBUG,
     		    "Failed to stat file %s, (returnCode = 0x%x, reasonCode = 0x%x)\n",
             oldAbsolutePath, returnCode, reasonCode);
-    return -1;
+    return HTTP_FILE_SERVICE_NOT_FOUND;
   }
   
   status = fileInfo(newAbsolutePath, &info, &returnCode, &reasonCode);
   if (status == 0 && !forceCopy) {
-    return -1;
+    zowelog(NULL, LOG_COMP_RESTFILE, ZOWE_LOG_DEBUG,
+            "File already exists %s\n", newAbsolutePath);    
+    return HTTP_FILE_SERVICE_ALREADY_EXISTS;
   }
 
   status = fileCopy(oldAbsolutePath, newAbsolutePath, &returnCode, &reasonCode);
   if (status == -1) {
     zowelog(NULL, LOG_COMP_RESTFILE, ZOWE_LOG_DEBUG,
             "Failed to copy file %s, (returnCode = 0x%x, reasonCode = 0x%x)\n",
             oldAbsolutePath, returnCode, reasonCode);
-    return -1;
+    if(returnCode == BPX_EACCES){
```

Comment:
Please fix formatting. Conditionals should look like this:
```C
if (condition) {
  consequent;
} else {
  alternate;
}
```

Reply:
Fixed formatting as per comment.